compiler/one-cmds/one-quantize

```diff
@@ -250,11 +250,16 @@ def _get_parser():
 
     # ampq_algorithm
     ampq_quant_group.add_argument(
-        '--ampq_algorithm', type=str, help='type of algorithm (bisection)')
+        '--ampq_algorithm', type=str, help='type of algorithm (bisection, pattern)')
 
     ampq_quant_group.add_argument(
         '--bisection_type', type=str, help="one of 'auto', 'i16_front', 'i16_back'")
 
+    ampq_quant_group.add_argument(
+        '--mixed_layernorm',
```

Comment:
Could you rename this? `mixed_layernorm` seems to ambiguous.

```suggestion
        '--u8_layernorm_with_s16_variance',
```

Reply:
> Could you rename this? `mixed_layernorm` seems ambiguous.

ok.